containers/Dockerfile.EESSI-build-node-debian10

```diff
@@ -1,5 +1,6 @@
 ARG cvmfsversion=2.8.2
 ARG awscliversion=1.20.39
```

Comment:
Also bump to latest? (see https://pypi.org/project/awscli/)

Reply:
Done!